notebooks/Datasets_1_Prepare_PubChem_Datasets.ipynb

```diff
@@ -99,10 +99,19 @@
    "metadata": {},
```

Comment:
<p>A much more straightforward way:</p><pre class="ql-syntax" spellcheck="false">if len(cid_to_pubchem) != len(df):
&nbsp;&nbsp;for i, row in tqdm(df.iterrows(), total=len(df)):
&nbsp;&nbsp;&nbsp;&nbsp;if row.cid in cid_to_pubchem:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;continue
&nbsp;&nbsp;&nbsp;&nbsp;compound = Compound.from_cid(row.cid)
&nbsp;&nbsp;&nbsp;&nbsp;cid_to_pubchem[row.cid] = compound
assert len(cid_to_pubchem) == len(df)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
done